### PR TITLE
Add benchmark for connection fields

### PR DESCRIPTION
**Pull request by jnak**
PR preview

### CODE REVIEW

**Comment by Nabellaleen** [comment]
Nice ! I already imagine how a Github Action/Bot could display the benchmark comparison with the previous version ;)
But I think this should be for another PR ;)

**Review by Nabellaleen — APPROVED** [review]

**Comment by jnak** [comment]
@Nabellaleen Thanks for reviewing! Though I can't merge it yet in the `graphene-sqlalchemy` repo because I need to send a PR once I merge https://github.com/graphql-python/graphene-sqlalchemy/pull/254 in `master`.

**Pull request closed without merging**
